### PR TITLE
update transaction cost calculations

### PR DESCRIPTION
**Pull request by rudigiesler**
Update the transaction cost calculations to include calculating the cost of the time length of a session.

### CODE REVIEW

File: go/billing/models.py [hunk 1]
```diff
@@ -147,6 +149,11 @@ def calculate_credit_cost(cls, message_cost, storage_cost, markup_percent,
         base_cost = message_cost + storage_cost
         if session_created:
             base_cost += session_cost
+        if session_length and session_unit_length and session_unit_cost:
+            units = (
+                Decimal(session_length)/Decimal(session_unit_length)
```
Comment (justinvdm) [inline go/billing/models.py]: Sorry to bikeshed -- pep8 will probably complain about the `/` not having spaces around it.

Comment (justinvdm) [inline go/billing/models.py]: Separate, more relevant comment: should we be doing the `Decimal()` wrapping we are doing here? We seem to assume the other params are decimals, so maybe we should assume these two are decimals also.

Comment (rudigiesler) [inline go/billing/models.py]: Good catch on the pep8, I wonder why my editor didn't highlight it. All change to assume we're getting a Decimal.

File: go/billing/models.py [hunk 2]
```diff
@@ -147,6 +149,11 @@ def calculate_credit_cost(cls, message_cost, storage_cost, markup_percent,
         base_cost = message_cost + storage_cost
         if session_created:
             base_cost += session_cost
+        if session_length and session_unit_length and session_unit_cost:
+            units = (
+                session_length / session_unit_length
+                ).to_integral_exact(rounding=ROUND_CEILING)
```
Comment (justinvdm) [inline go/billing/models.py:155]: Based on the slack #vumi-mgmt discussion, we should ceil the value here, I think.

**Comment by justinvdm** [comment]
One more minor comment, otherwise looks good.

File: go/billing/models.py (hunk 2 above)
Comment (justinvdm) [inline go/billing/models.py:155]: Doh, misread, we already are using ceil, sorry.

**Comment by justinvdm** [comment]
:+1: when travis is happy.

**Comment by hodgestar** [comment]
Belated :+1:. :)

**Pull request merged**